AlfrescoApp/View Controllers/Accounts View Controllers/AccountsViewController.m

```diff
@@ -39,10 +39,29 @@
 
 @interface AccountsViewController ()
 @property (nonatomic, assign) NSInteger expandedSection;
+@property (nonatomic, strong) NSDictionary *configuration;
+@property (nonatomic, assign) BOOL canAddAccounts;
+@property (nonatomic, assign) BOOL canRemoveAccounts;
 @end
 
 @implementation AccountsViewController
 
+- (instancetype)initWithConfiguration:(NSDictionary *)configuration session:(id<AlfrescoSession>)session
+{
+    self = [self initWithSession:session];
+    if (self)
+    {
+        self.configuration = configuration;
+        NSNumber *canAddAccounts = configuration[kAppConfigurationCanAddAccountsKey];
+        self.canAddAccounts = (canAddAccounts) ? canAddAccounts.boolValue : YES;
```

Comment:
Do the parentheses here have special meaning - checking for nil rather than the variable value? I can't quite work out how a `@NO` value works otherwise...